config.test.yaml

```diff
@@ -64,6 +54,27 @@ templates:
           internal:
             - /^(?:::ffff:)?(?:10|127)\./
 
+    x-modules:
+      - name: mobileapps-public
+        path: specs/mediawiki/v1/mobileapps
+        type: spec
+        options:
+          host: http://appservice.wmflabs.org
+
+      - name: graphoid
+        path: mods/restbase/graphoid_v1
```

Comment:
Yeah, I was thinking about moving to a `/v1/` (or `/rest_v1`) vs `/sys/` distinction as well. Less sure about the `wikimedia` prefix. Perhaps no prefix at all?